pkg/operator/controller/certrotation_test.go

```diff
@@ -32,6 +32,25 @@ var secretNames = []string{
 	"cdi-uploadserver-signer", "cdi-uploadserver-client-signer", "cdi-uploadserver-client-cert",
```

Comment:
do you mind explaining in the commit message why? better the needing to get inside the link and also whats linked does explain exactly the issue.

Reply:
it's a rabbit hole, have to follow the readouts. but mostly its the fake client not supporting resourceVersion